ios/src/TGMapViewController.mm

```diff
@@ -9,11 +9,13 @@
 
 #import "TGMapViewController.h"
 #include "platform_ios.h"
+#include "data/propertyItem.h"
 #include "tangram.h"
 
 @interface TGMapViewController ()
 
-@property (nullable, strong, nonatomic) EAGLContext *context;
+@property (nullable, nonatomic) NSString* scenePath;
```

Comment:
We _could_ use `NSURL` here and just get the string representation for calls into C++. This variable often refers to files inside the current application bundle, is that easy with `NSURL`?